src/Lookup/EntityRetrievingTermLookup.php

```diff
@@ -38,11 +39,20 @@ public function __construct( EntityLookup $entityLookup ) {
 	 * @param EntityId $entityId
 	 * @param string $languageCode
 	 *
-	 * @return string
+	 * @return string|null
+	 * @throws TermLookupException
 	 */
 	public function getLabel( EntityId $entityId, $languageCode ) {
-		$labels = $this->getFingerprint( $entityId )->getLabels();
-		return $labels->getByLanguage( $languageCode )->getText();
+		$fingerprint = $this->getFingerprint( $entityId, array( $languageCode ) );
+
+		/** @var Fingerprint $fingerprint */
+		$labels = $fingerprint->getLabels();
+
+		try{
```

Comment:
Fixed in https://github.com/wmde/WikibaseDataModelServices/commit/180b24fc3e2be759d61a9098d76f23c7d94a8c45